src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculatable {
+    private Color color;
+
+    public Figure(String name, Color color) {
```

Comment:
```suggestion
    public Figure(Color color) {
```
you don't need this field for figures, remove it